integration-templates/calendly/tests/calendly-event-invitees.test.ts

```diff
@@ -0,0 +1,114 @@
+import { vi, expect, it, describe } from 'vitest';
+import type { NangoSync } from '../models.js';
+
+import fetchData from '../calendly/syncs/event-invitees.js';
```

Comment:
paths could be generated relative to template `../syncs/event-invitees.js`

Reply:
Had to restructure tests some, but this has now been updated.